frontend/packages/schema-model/src/lib/mutations/ui-schema-reducers.test.ts

```diff
@@ -367,6 +367,32 @@ describe('ui-schema-reducers', () => {
       expect(updatedNode.fieldType).toEqual(stringNodeMock.fieldType);
       expect(updatedNode.isArray).toBe(true);
     });
+
+    it("Updates the childrens' pointers", () => {
+      const { pointer } = parentNodeMock;
+      result = toggleArrayField(uiSchemaMock, pointer);
+      const updatedNode = getNodeByPointer(result, pointer);
+      expect(updatedNode.children.length).toEqual(parentNodeMock.children.length);
+      updatedNode.children.forEach((childPointer) => {
+        expect(childPointer.startsWith(`${pointer}/${Keyword.Items}`)).toBe(true);
+      });
+      getChildNodesByPointer(result, pointer).forEach((childNode) => {
+        expect(childNode.pointer.startsWith(`${pointer}/${Keyword.Items}`)).toBe(true);
+      });
+    });
+
+    it("should  not update the children's pointers with /Items/ when isArray OFF", () => {
+      const { pointer } = parentNodeMock;
+      result = toggleArrayField(uiSchemaMock, pointer);
```

Comment:
You need to use different test data in order to hit the `else` block. F.ex.
```suggestion
      result = toggleArrayField({ ...uiSchemaMock, isArray: false }, pointer);
```